### ISSUE TITLE
Design permission group and access model for Unity Catalog and Workflow

### ISSUE THREAD

**Issue by danielgils** [WB-DECIS/tidysdmx#66]
📃 **Context**
Need a better understanding on what role should have what level of accessibility to set up the workflow 

-  What happens when the same person has two different roles in multiple pipelines?

✅ **Acceptance criteria**

📦 **Deliverables**

- [ ] Clarify the roles and responsibility in pipeline work 
- [ ] Set up user group for each role
- [ ] Assign default access for each user group
